playbooks/support/scheduling.md

```diff
@@ -107,5 +78,28 @@ To override a shift:
 
    <img width=500 src="images/opsgenie-override-2.png" />
 
-[schedule]: https://artsy.app.opsgenie.com/teams/dashboard/ee381004-a72e-42ef-a733-b350d6693c6c/main
-[artsy-cli]: https://github.com/artsy/artsy-cli
+## **Adding Engineers**
+
+Generally, new engineers should be added to the end of the schedule rotation associated with their time zone. To do this: 
+
+1. Navigate to the [On-Call Schedule](https://artsy.app.opsgenie.com/teams/dashboard/ee381004-a72e-42ef-a733-b350d6693c6c/main) and edit the engineer's respective rotation.
+
+   <img width=500 src="images/opsgenie-addengs-1.png" />
+   <img width=500 src="images/opsgenie-addengs-2.png" />
+
+2. Add the engineer to the end of the participants list. Their position in the rotation will match their position in the participants list on the rotation edit page, for example:
+
+Considerations when adding a new engineer:
+
+- You should aim to avoid pairings with two new engineers and you can place the engineer earlier in the rotation to avoid this.
+- **NOTE: If you add engineers anywhere other than the end of the rotation, the schedule of the engineers who have not completed their shifts will be modified, as the order of the participants matters. You should therefore update the affected engineers to avoid scheduling conflicts.**
+- A new hire should be given 60 days after their start date to get the necessary context before having an on-call shift.
```

Comment:
Should managers add engineers to the rotation right when they join (but ensure that they aren't on the schedule for 60 days), or should they wait until they've been at Artsy for 60 days and then add them to the rotation?

I'm not sure if it matters, but I'm sure that people will ask. 😄

Reply:
This is a good call out. My initial thinking was that they should do it right away. But if we are within the 60-days  or a rotation completing adding them to the end would violate this guidance. I will update to have managers delay adding until a new rotation has started if within the 60 days.